app-with-redux-toolkit/src/redux/services.ts

```diff
@@ -0,0 +1,21 @@
+import { customDelay } from "./customFunctions/customDelay";
+
+let accountBalance = 0;
+
+export const changeAccountState = async (
+  deltaAmount: number,
+  interestRateCalculation = false
+) => {
+  const randomError = Math.random() <= 0.75;
+
+  await customDelay(2000);
+  if (randomError) {
+    return (accountBalance = Math.round(accountBalance + deltaAmount));
```

Comment:
hele upřímnš se mi nelíbí, že v jednom řádku assignuješ nějakou globální proměnnou a zároveň vracíš hodnotu, nemůžeš to udělat na 2, aby se to lépe četlo?